db/schema.sql

```diff
@@ -65,8 +73,48 @@ CREATE TABLE IF NOT EXISTS team_student_weekly_contributions (
   num_commits integer NOT NULL
 );
 
+CREATE TABLE IF NOT EXISTS student_attendance (
+    id SERIAL PRIMARY KEY,
+    enrollment_id integer REFERENCES enrollments,
+    zoom_id text NOT NULL,
+    user_id integer NOT NULL,
+    user_name text NOT NULL,
+    device text,
+    LOCATION text,
+    data_center text,
+    join_time date NOT NULL,
+    leave_time date NOT NULL,
+    recording boolean,
+    pc_name text,
+    first_name text,
+    last_name text,
+    room text NOT NULL,
+    created_at timestamptz NOT NULL DEFAULT now()
+);
 
+CREATE TABLE IF NOT EXISTS student_absences (
+    id SERIAL PRIMARY KEY,
+    enrollment_id integer REFERENCES enrollments,
+    first_name text,
+    last_name text,
+    user_name text NOT NULL,
+    cohort text,
+    type text,
+    is_absent boolean,
+    is_late boolean,
+    data_center text,
+    notes_id integer REFERENCES students_staff_comments,
+    is_excused boolean,
+    points NOT NULL DEFAULT 0,
+    created_at timestamptz NOT NULL DEFAULT now()
+);
 
-
-
+CREATE TABLE IF NOT EXISTS students_staff_comments (
```

Comment:
All comments will be saved in this table. If the comments are related to absences it will be displayed in the attendance/absences view. 
All comments for a student will be visible in the student lookup section under admin view.